addons/sale_expense/views/sale_order_views.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="sale_order_form_view_inherit" model="ir.ui.view">
```

Comment:
CompletelyUnnecessaryRemark: With Yti we thought the name did not really required the "_inherit" suffix, as its complete xml_id would be "sale_expense.sale_order_view_form", already telling it is an inheritance. By the way, sale_order_view_form (bike shed color).

Reply:
CompletelyUnnecessaryResponse: I like this suffix to tell it's a inherited view. I would removed it if the module redefined a new PRIMARY view by extended an exsting one. Guidelines toussa toussa. We should definitely update them ....